test/integration/targets/nxos_acl_interfaces/tests/cli/deleted.yml

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
deleted.yml should test:

(a) Deleting a single ACL from an interface.
(b) Deleting all ACLs of a particular AFI from an interface.
(c) Deleting all ACLs from an interface.
(d) Deleting all ACLs from all interfaces on the appliance.